### ISSUE TITLE
Mac(Yosemite) cannot compile because QT_STATIC_CONST is unknown type name

### ISSUE THREAD

**Issue by zhang356** [marsyas/marsyas#43]
They following is what terminal gave me after I do "make -j3"
Obviously QT_STATIC_CONT is an unknown name which causes the error. I looked it up, it's because the linked header file the QT_STATIC_CONST in "qglobal.h" were removed. 

So what should I do?

[ 69%] Built target sound2sound
[ 69%] Built target ibt
[ 69%] /Users/Frank/Desktop/marsyas/src/otherlibs/qwt/qwt_transform.h:111:5: error: 
      unknown type name 'QT_STATIC_CONST'
    QT_STATIC_CONST double LogMax;
    ^
/Users/Frank/Desktop/marsyas/src/otherlibs/qwt/qwt_transform.h:111:21: error: 
      expected member name or ';' after declaration specifiers
    QT_STATIC_CONST double LogMax;
    ~~~~~~~~~~~~~~~ ^
Built target pitchdtw
[ 69%] Built target pitchdtw_pair

**Comment by nok** [comment]
Hello all, here is a short workaround. So far I didn't run any executables, but the build was successful.

`/src/otherlibs/qwt/qwt_transform.h`:

```cpp
// old
QT_STATIC_CONST double LogMin;
QT_STATIC_CONST double LogMax;

// new
static const double LogMin;
static const double LogMax;
```

`/src/otherlibs/qwt/qwt_transform.cpp`:

```cpp
// old
QT_STATIC_CONST_IMPL double QwtLogTransform::LogMin = 1.0e-150;
QT_STATIC_CONST_IMPL double QwtLogTransform::LogMax = 1.0e150;

// new
const double QwtLogTransform::LogMin = 1.0e-150;
const double QwtLogTransform::LogMax = 1.0e150;
```

Finally I didn't create a pull request, because I'm not familiar in Qt.

Happy coding,
Darius

**Comment by jleben** [comment]
Fixed in commit 596145b6ed1b0db9ed93130b330a91998760c6e0
